device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/twin/DirectMethodResponse.java

```diff
@@ -19,7 +19,7 @@ public int getStatus()
         return status;
     }
 
-    public String getResponseMessage()
+    public Object getResponseMessage()
```

Comment:
rename this to getPayload to match the other similar classes

Reply:
And rename the instance variable to be called payload instead of responseMessage